tmux/demos/medical/Kitware-R18.yml

```
@@ -64,21 +67,17 @@ windows:
         -p pose_config:=${ANGEL_WORKSPACE_DIR}/python-tpl/TCN_HPL/tcn_hpl/data/utils/pose_generation/configs/ViTPose_base_medic_casualty_256x192.py
         -p cuda_device_id:=0
 
-    - objects_hands_detection: ros2 run angel_system_nodes object_hand_detector --ros-args 
-        -r __ns:=${ROS_NAMESPACE}
-        -p image_topic:=image_0 
-        -p det_topic:=ObjectDetections2d 
-        -p net_checkpoint:=${MODEL_DIR}/object_detector/r18_det.pt
-        -p hand_net_checkpoint:=${MODEL_DIR}/object_detector/hands_model.pt
-
-    - simple_2d_overlay: ros2 run angel_utils Simple2dDetectionOverlay --ros-args
-        -r __ns:=${ROS_NAMESPACE}
-        -p topic_input_images:=image_0
-        -p topic_input_det_2d:=ObjectDetections2d
-        -p topic_output_images:=pv_image_detections_2d
-        -p filter_top_k:=5
+    - object_detector:
+        layout: even-vertical
+        panes:
+            - objects_hands_detection: ros2 run angel_system_nodes object_hand_detector --ros-args 
```

Nitpick: Could we call this 
```suggestion
            - objects_and_hands_detection: ros2 run angel_system_nodes object_hand_detector --ros-args 
```

I keep reading this incorrectly and think that this is only detecting hands.